docs/copy-secrets-to-vault.md

```diff
@@ -0,0 +1,58 @@
+# `copy-secrets-to-vault`
+This `vault-radar` command is used for copying unVaulted secrets discovered in a supported scan into HashiCorp Vault.
```

Comment:
We updated the terminology in the scan output to `managed`, so we should probably use `managed`/`unmanaged` across this doc

Reply:
ah okay thanks! f0b792d